### ISSUE TITLE
region_size change

### ISSUE THREAD

**Issue by MathieuCliche** [TensorBox/TensorBox#209]
Is it possible to change the region_size so something else than 32?  If so what are the constraints ?

So far everything I tried gives me shapes errors.  

Thanks!

**Comment by Russell91** [comment]
Yeah, it is possible, but requires fairly intimate understanding of the rezoom. So probably best to just keep this as it is.
